app/views/layouts/application.html.erb

```diff
@@ -46,7 +46,11 @@
           <span class="icon">
             <i class="fa fa-search"></i>
           </span>
-          <input type="search" id="search" placeholder="Search..."/>
+          <!-- <input type="search" id="search" placeholder="Search..."/> -->
```

Comment:
get rid of commented out code! it can actually break stuff in ERB, since the `<!--` comment indicators do not always get interpreted properly!